cli/_posts/2017-01-02-syncing-files-through-runnable-c-l-i.md

```diff
@@ -7,13 +7,18 @@ description: Syncing Files through Runnable CLI
 
 Instead of uploading files through the UI on [runnable.io](http://runnable.io), our runnable-cli enables you to upload files to your repository directly from your command line.
 
-    Usage:
-      runnable upload <file>
+---
+
+### Usage
+`runnable upload <file>`
+
+Use this command to upload a file to the repository.
+The file will be placed at the root of the repository folder in the container.
```

Comment:
Since single newlines (without an empty newline above them) don't create a new paragraph, let's try to avoid using them. I think in this case it's cool for these two sentences to be in the same paragraph, so let's just have a space between them instead of a newline.